packages/evolution-common/src/services/interviews/interview.ts

```diff
@@ -36,19 +36,19 @@ export const auditInterview = function (
     const response = correctedResponse ? correctedResponse : originalResponse;
     const household = _get(response, 'household', {});
     const home = _get(response, 'home', {});
-    const validatedInterview = _cloneDeep(interview);
-    validatedInterview.response = response;
+    const correctedInterview = _cloneDeep(interview);
+    correctedInterview.response = response;
     // TODO Rethink this, the _response field does not exist on the type and should not have to exist
-    (validatedInterview as any)._response = originalResponse;
-    const persons = surveyProjectHelper.getPersons(validatedInterview, false);
-    const personsArray = surveyProjectHelper.getPersons(validatedInterview, true);
+    (correctedInterview as any)._response = originalResponse;
```

Comment:
Setting response to either correctedResponse or originalResponse and _reponse to originalResponse... what can go wrong!!??

Also the interview is cloned, but not the responses that we assigned to this cloned interview. WHAT can go wrong!!??

Reply:
added a fixme